File: cpp_codes/src/fizz_buzz.cpp
#include <iostream>
#include <string>

void fizz_buzz(int num) {
    std::string answer = "";
    if(num % 3 == 0) answer += "Fizz";
    if(num % 5 == 0) answer += "Buzz";
    if(num % 3 != 0 && num % 5 != 0) answer += std::to_string(num);
    std::cout << answer << std::endl;
}

int main() {
    fizz_buzz(3);
    fizz_buzz(5);
    fizz_buzz(15);
    fizz_buzz(1);
  return 0;
}

File: java_codes/src/FizzBuzz.java
public class FizzBuzz {
    static void solution(int num) {
        String answer = "";
        if(num % 3 == 0) answer += "Fizz";
        if(num % 5 == 0) answer += "Buzz";
        if(num % 3 != 0 && num % 5 != 0) answer += Integer.toString(num);
        System.out.println(answer);
    }
    public static void main(String[] args) throws Exception {
        solution(3);
        solution(5);
        solution(15);
        solution(1);
    }
}
